dependencies.yaml

```diff
@@ -100,7 +100,7 @@ dependencies:
       match: go \d+.\d+
 
   - name: "golang: after kubernetes/kubernetes update"
-    version: 1.17.5
+    version: 1.17.6
```

Comment:
we should bump this after the k/k go updates are merged

Reply:
Thanks!